armips/include/macros.s

```diff
@@ -15,7 +15,7 @@
 .endmacro
 
 .macro learnset,move,level
-.if move <= MOVE_FUSION_BOLT
+.if move < NUM_OF_MOVES
```

Comment:
This is to remove the 562 cap that was in place due to the relevant files not being created yet past this number